docs/current/sdk/elixir/snippets/get-started/step4/a_project.test.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Mix.Tasks.AProject.Test do
+  use Mix.Task
+
+  @impl Mix.Task
+  def run(_args) do
+    Application.ensure_all_started(:dagger)
+
+    client = Dagger.connect!()
+
+    project =
+      client
+      |> Dagger.Client.host()
+      |> Dagger.Host.directory(".", exclude: ["deps", "_build"])
+
+    # highlight-start
+    [
+      {"1.14.5", "25.3.2.5"},
+      {"1.15.4", "25.3.2.5"},
+      {"1.15.4", "26.0.2"}
```

Comment:
Is it worth the user's time in showing 3 examples instead of 2?

To prove the multiple versions running in parallel point, I think that it would be enough to run the same Elixir version against multiple Erlang versions.

Reply:
I modified it to use 2 Elixir versions instead. 🙇

Reply:
Thank you!

I've made this more realistic. I was imagining someone that is on Elixir `v1.14.4` and wants to see if the app will work fine on `v1.15.0-rc.2` with Erlang `v26`.

Also switched to Alpine since it's 40-50MB smaller. The quicker these getting started guides run, the better. The main goal is to show how all the things fit together.

FWIW, Ubuntu Bionic (18.04 LTS) is pretty old at this point - ESM right now. Focal (20.04 LTS) or Jammy (22.04) are the currently supported Ubuntu LTS versions.